### PR TITLE
Code review comments in-line.

### PR DESCRIPTION
**Pull request by dajtxx**
github pull request review only allows omments to be put on modified lines.

The comments should all start with //z to make them easy to find.

This branch should not be merged to master.

### CODE REVIEW

File: Firmware/Static-weighstation/v0.4/Teensy3.6/main/sensors.h [hunk 1]
```diff
@@ -98,6 +102,7 @@ class WeighStation {
 
 };
 
+//z LOL, static all the things.
```
Comment (HarveyBates) [inline Firmware/Static-weighstation/v0.4/Teensy3.6/main/sensors.h:105]: haha when in doubt make it static 👍

File: Firmware/Static-weighstation/v0.4/Teensy3.6/main/sensors.h [hunk 2]
```diff
@@ -84,6 +87,7 @@ class WeighStation {
   // Scan each of the scales and capture any animal weights
   void scan();
 
+//z Maybe uint8_t* here & below.
```
Comment (HarveyBates) [inline Firmware/Static-weighstation/v0.4/Teensy3.6/main/sensors.h:90]: yea I'll tidy this up a bit

File: Firmware/Static-weighstation/v0.4/Teensy3.6/main/lora.cpp [hunk 3]
```diff
@@ -206,7 +208,10 @@ void onEvent (ev_t ev) {
   }
 }
 
-
+//z I used this to check the state of the LoRa stack:
+//z return LMIC.opmode & (OP_JOINING | OP_TXRXPEND | OP_SHUTDOWN) ? false : true;
+//z That might not be great because I would see occasional huge delays in sending
+//z messages, but it might be worth looking at that variable.
```
Comment (HarveyBates) [inline Firmware/Static-weighstation/v0.4/Teensy3.6/main/lora.cpp:214]: ill look into it further, anything to improve the lora side of things 👍

**Review by HarveyBates — COMMENTED** [review]
thank you very much for going through this 👍  helps out a lot as I haven't had any of my work reviewed before so its hard to know where I am going wrong

**Comment by dajtxx** [comment]
It’s pretty good you’ve got it all working, that’s not trivial code.

If you tinker with that array of hx711 objects, the initialisation will be something like static HX711[] = { &scaleOne, &scaleTwo, &scaleThree } and all the calls will need . changed to -> like HX711[i]->something();

For loop indexes are another odd one. I googled it earlier this year because you’d expect them to be unsigned, but apparently lots of people use signed indexes! Not sure how that works out. If it’s working it’s probably best to leave it as is.

It’s good you have lots of comments already.

**Pull request closed without merging**